src/HOCs/imgixProvider.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext, createContext } from 'react'
+
+const ImgixContext = createContext()
+
+/**
+ * `useImgixContext()` tries to invoke `React.useContext()`. If context is 
+ * `undefined`, we know that the context is being accessed outside of a 
+ * provider, and therefore throw an `Error`.
+ * @returns Hook that gets context value from the nearest`ImgixProvider`.
```

Comment:
```suggestion
 * `useImgixContext()` tries to invoke `React.useContext()`. If no context
 * is available, this function returns `undefined`. 
 * @returns The context defined by the closest parent `ImgixProvider`.
```

I verified that `React.useContext` does not throw an error if context is not passed

Reply:
Ah thanks, this jsdoc got outdated since the context check was moved outside of this function.